src/backend/joanie/tests/test_admin_course.py

```diff
@@ -29,6 +32,22 @@ def test_admin_course_use_translatable_change_form_with_actions_template(self):
         with self.assertTemplateUsed(
             "joanie/admin/translatable_change_form_with_actions.html"
         ):
-            self.client.get(
+            response = self.client.get(
                 reverse("admin:core_course_change", args=(course.pk,)),
             )
+
+        self.assertEqual(response.status_code, 200)
+        self.assertContains(response, course.title)
+
+        html = lxml.html.fromstring(response.content)
+
+        # Django parler tabs should be displayed
+        parler_tabs = html.cssselect(".parler-language-tabs span")
+        self.assertEqual(len(parler_tabs), len(settings.LANGUAGES))
```

Comment:
I would just use an override to set the languages tab to be sure it's set and compare it to its actual length